nucleus/payara-modules/healthcheck-core/src/main/java/fish/payara/nucleus/healthcheck/preliminary/HoggingThreadsHealthCheck.java

```diff
@@ -198,7 +198,7 @@ public void collect(MonitoringWatchCollector collector) {
             .red(1, -2, false, null, null, false);
     }
 
-    private void acceptHoggingThreads(Map<Long, ThreadCpuTimeRecord> recordsById, HoggingThreadConsumer consumer) {
+    private synchronized void acceptHoggingThreads(Map<Long, ThreadCpuTimeRecord> recordsById, HoggingThreadConsumer consumer) {
```

Comment:
Why this one was changed to synchronized?
I did not see a relation with the issue.

Reply:
Accident, I will remove.